client/src/components/organisms/TimeLine/TimeLine.jsx

```diff
@@ -65,68 +68,76 @@ const TimeLine = () => {
               greens. Fennel quandong potato watercress ricebean swiss chard
               garbanzo. Endive daikon brussels sprout lotus root silver beet
               epazote melon shallot.
-            </p>
-          </div>
-        </li>
-        <li>
-          <time className="cbp_tmtime" dateTime="2013-04-16 21:30">
-            <span>4/16/13</span> <span>21:30</span>
-          </time>
-          <div className="cbp_tmicon cbp_tmicon-earth" />
-          <div className="cbp_tmlabel">
-            <h2>Courgette daikon</h2>
-            <p>
+          </p>
+        </div>
+      </li>
+      <li>
+        <time className="cbp_tmtime" dateTime="2013-04-16 21:30">
+          <span>4/16/13</span>
+          {' '}
+          <span>21:30</span>
+        </time>
+        <div className="cbp_tmicon cbp_tmicon-earth" />
+        <div className="cbp_tmlabel">
+          <h2>Courgette daikon</h2>
+          <p>
               Parsley amaranth tigernut silver beet maize fennel spinach.
               Ricebean black-eyed pea maize scallion green bean spinach cabbage
               jícama bell pepper carrot onion corn plantain garbanzo. Sierra
               leone bologi komatsuna celery peanut swiss chard silver beet
               squash dandelion maize chicory burdock tatsoi dulse radish wakame
               beetroot.
-            </p>
-          </div>
-        </li>
-        <li>
-          <time className="cbp_tmtime" dateTime="2013-04-17 12:11">
-            <span>4/17/13</span> <span>12:11</span>
-          </time>
-          <div className="cbp_tmicon cbp_tmicon-screen" />
-          <div className="cbp_tmlabel">
-            <h2>Greens radish arugula</h2>
-            <p>
+          </p>
+        </div>
+      </li>
+      <li>
+        <time className="cbp_tmtime" dateTime="2013-04-17 12:11">
+          <span>4/17/13</span>
+          {' '}
+          <span>12:11</span>
+        </time>
+        <div className="cbp_tmicon cbp_tmicon-screen" />
+        <div className="cbp_tmlabel">
+          <h2>Greens radish arugula</h2>
+          <p>
               Caulie dandelion maize lentil collard greens radish arugula sweet
               pepper water spinach kombu courgette lettuce. Celery coriander
               bitterleaf epazote radicchio shallot winter purslane collard
               greens spring onion squash lentil. Artichoke salad bamboo shoot
               black-eyed pea brussels sprout garlic kohlrabi.
-            </p>
-          </div>
-        </li>
-        <li>
-          <time className="cbp_tmtime" dateTime="2013-04-18 09:56">
-            <span>4/18/13</span> <span>09:56</span>
-          </time>
-          <div className="cbp_tmicon cbp_tmicon-phone" />
-          <div className="cbp_tmlabel">
-            <h2>Sprout garlic kohlrabi</h2>
-            <p>
+          </p>
+        </div>
+      </li>
+      <li>
+        <time className="cbp_tmtime" dateTime="2013-04-18 09:56">
+          <span>4/18/13</span>
+          {' '}
```

Comment:
What with these spaces? You can use `\u00A0`, also why two separate `spans`? you can use single span.

Reply:
I could not get such an output using anything else. 

![Screenshot 2019-09-30 at 6 00 57 PM](https://user-images.githubusercontent.com/15229752/65878725-50bdb200-e3ac-11e9-95ce-68427cb0ec1a.png)